system/MenuButton.tsx

```diff
@@ -27,7 +24,11 @@ export default function MenuButton(props) {
 
   return (
     <div className={styles.body}>
-      <div ref={hamburgerRef} className={styles.hamburger} onClick={toggleModal}>
+      <div
+        ref={hamburgerRef}
+        className={`${styles.hamburger} ${menuActive ? styles.active : ''}`}
```

Comment:
the team should use `Utilities.classNames()` but i am guilty of doing this in the past as well, so i won't hold up the review for this

Reply:
Fixed, thank you!